examples/angular/src/pages/ui/components/authenticator/custom-slots/custom-slots.component.html

```diff
@@ -0,0 +1,77 @@
+<amplify-authenticator>
+  <ng-template amplifySlot="header">
+    <div style="padding: var(--amplify-space-large); text-align: center">
+      <img
+        class="amplify-image"
+        alt="Amplify logo"
+        src="https://docs.amplify.aws/assets/logo-dark.svg"
+      />
+    </div>
+  </ng-template>
+
+  <ng-template amplifySlot="sign-in-header">
+    <h3
+      class="amplify-heading"
+      style="padding: var(--amplify-space-xl) 0 0 var(--amplify-space-xl)"
+    >
+      Sign in to your account
+    </h3>
+  </ng-template>
+
+  <ng-template amplifySlot="sign-in-footer" let-authenticator="authenticator">
+    <div style="text-align: center">
+      <button
+        (click)="authenticator.toResetPassword()"
+        class="amplify-button amplify-field-group__control"
+        data-fullwidth="false"
+        data-size="small"
+        data-variation="link"
+        type="button"
+        style="font-weight: normal"
+      >
+        Reset Password
+      </button>
+    </div>
+  </ng-template>
+
+  <ng-template amplifySlot="sign-up-header">
+    <h3
+      class="amplify-heading"
+      style="padding: var(--amplify-space-xl) 0 0 var(--amplify-space-xl)"
+    >
+      Create a new account
+    </h3>
+  </ng-template>
+
+  <ng-template amplifySlot="sign-up-footer" let-toSignIn="toSignIn">
+    <div style="text-align: center">
+      <button
+        class="amplify-button amplify-field-group__control"
```

Comment:
Just like `toResetPassword`, I need help getting this `(click)` functionality wired up.  Ideally, using `authenticator.toSignIn()`.